test_evenorodd.py

```diff
@@ -1,4 +1,10 @@
 import evenorodd
 
-def test_even_or_odd():
-    assert(evenorodd.even_or_odd(2) == "even")
\ No newline at end of file
+def test_even():
+    assert(evenorodd.even_or_odd(2) == "even")
+
+def test_odd():
+    assert(evenorodd.even_or_odd(71) == "odd")
+
+def test_even_large_number():
```

Comment:
Looks good - could be worth a test for 0 too - that feels like it might be a place things could go wrong.